pkg/imgutil/push/push.go

```diff
@@ -48,7 +48,7 @@ func Push(ctx context.Context, client *containerd.Client, resolver remotes.Resol
 	}
 	desc := img.Target
 
-	ongoing := newPushJobs(commands.PushTracker)
+	ongoing := newPushJobs(dockerconfigresolver.PushTracker)
```

Comment:
Could you explain how this fixes the issue

Reply:
`commands.PushTracker` and [`tracker in resolver`](https://github.com/containerd/containerd/blob/v1.6.0-rc.4/remotes/docker/resolver.go#L137) are not the same object, they are not shared.